meson_options.txt

```diff
@@ -40,6 +40,7 @@ option('elogind', type : 'boolean', value : false, description : 'enable elogind
 option('tests', type : 'boolean', value : true, description : 'enable tests')
 option('soup_session_compat', type : 'boolean', value : true, description : 'enable SoupSession runtime compatibility support')
 option('curl', type : 'boolean', value : true, description : 'enable libcurl support')
+option('lockdir', type: 'string', value: '', description: 'Directory for fwupdtool lock')
```

Comment:
does ChromeOS use something different to /run/lock?

Reply:
no, do you want me to remove the meson option and hard code it?

Reply:
I think that's best; I think /run/lock is pretty standard.

Reply:
Done

Reply:
Should I remove the FU_PATH_KIND_LOCKDIR too?

Reply:
I'm not sure. I think it makes sense to keep, in case the snap needs to override it. @superm1

Reply:
Isn't /var/run a symlink these days?

Reply:
It is, but I got complains about using it. Also, for the minijail sandbox to be effective, the lock should exist under /run/lock (or /var/run/lock) instead of plain /run so the tool does not have access to the whole contents of /run... crrev.com/c/2873506 for context